Assets/TheGrid/EnvironmentScripts/BasicTile.cs

```diff
@@ -2,22 +2,54 @@
 using System.Collections.Generic;
 using UnityEngine;
 
+[System.Serializable]
 public class BasicTile : MonoBehaviour {
 
 private int width = 10;
 private int widthInTiles = 1;
-private int height = 10;
-private int heightInTiles = 1;
+private int length = 10;
+private int lengthInTiles = 1;
 
 public bool buildable;
+public float rawHeight;
+public int planeLevel;
 public int xLoc;
 public int yLoc;
```

Comment:
Unity uses x and z for the coordinate plane you're referencing here. Y is used for vertical. May be worth changing so we don't get confused later.